### ISSUE TITLE
Can't remove the old feature_test video when run twice

### ISSUE THREAD

**Issue by runnersaw** [santosfamilyfoundation/SantosGUI#108]
Below is terminal output. This happens when I run a feature test video the second time on Windows 10.

```
Process Process-2:
Traceback (most recent call last):
  File "C:\Anaconda2\envs\santosgui\lib\site-packages\multiprocess\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Anaconda2\envs\santosgui\lib\site-packages\multiprocess\process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\svaughan\Documents\GitHub\SantosGUI\application\cloud_api.py", line 679, in _function
    ret = func(*args, **kwargs)
  File "C:\Users\svaughan\Documents\GitHub\SantosGUI\application\cloud_api.py", line 272, in getTestConfig
    self.writeToPath(r, file_path,file_name)
  File "C:\Users\svaughan\Documents\GitHub\SantosGUI\application\cloud_api.py", line 57, in writeToPath
    os.remove(path)
WindowsError: [Error 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\svaughan\\Documents\\SantosGUI\\project_dir\\demo\\feature_video\\feature_video.mp4'
```

**Comment by runnersaw** [comment]
Turns out the QMediaPlayer is keeping the file open on Windows. We have to close the file by giving our QMediaPlayer a null QMediaContent, before we can write to `feature_video.mp4`.

**Comment by runnersaw** [comment]
Resolved by #109
